### ISSUE TITLE
Please add Anonfilly

### ISSUE THREAD

**Issue by nixlad** [erkin/ponysay#296]
We need her!

**Comment by matthias4217** [comment]
Do you have any pixel-art version of it ? It would really help !

**Comment by nixlad** [comment]
> Do you have any pixel-art version of it ? It would really help !

Does this help?
<img width="218" alt="Screen Shot 2020-01-07 at 2 07 06 PM" src="https://user-images.githubusercontent.com/35383882/71921581-215ed400-3157-11ea-8f48-0f898e70f7c8.png">

**Comment by matthias4217** [comment]
Yes I think so. I don't have a lot of time at the moment, but I will do it probably later in the weekend.

**Comment by matthias4217** [comment]
![image](https://user-images.githubusercontent.com/17904729/72278502-722d6b80-3634-11ea-8355-6ded2bd87b3b.png)
So this is how it looks for now on my side. I probably will have to mess with the colour in a way, but I don't really know how to do it.

**Comment by nixlad** [comment]
perfect. 10/10

**Comment by matthias4217** [comment]
To finish it, I'd just need a few information about metadata. The description for all these can be found in the manual : https://github.com/erkin/ponysay/blob/master/ponysay.pdf (Section 6 : Pony metadata)
More precisely, I'd need 

- `BASED ON` (I believe it would be `(original)` if I understand correctly)
- `SOURCE` (I don't know where your screenshot comes from ^^)
- `MEDIA` (4chan ?)
- `LICENSE` (I dont know ?), and depending on it comes the metadata `FREE` if it's libre

Thank you for your help on this !

**Comment by nixlad** [comment]
Based on a popular OC representing anonymous posters.

Source screenshot comes from pony.town.

Media: not sure what this means.

As for license, screenshot is from a game where the source code is freely available under a CC BY-NC 4.0 non-commercial license.

Thank you very much.
